<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObservationTable</class>
 <widget class="QWidget" name="ObservationTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="levelOneTable">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="levelTwoTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>C</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="levelThreeTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>C</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>D</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>E</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="3" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="levelFourTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>C</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>D</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>E</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>F</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="3" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
    <property name="background">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
   <item row="4" column="3">
    <property name="text">
     <string/>
    </property>
    <property name="flags">
     <set>NoItemFlags</set>
    </property>
   </item>
  </widget>
  <zorder>levelTwoTable</zorder>
  <zorder>levelThreeTable</zorder>
  <zorder>levelFourTable</zorder>
  <zorder>levelOneTable</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
